<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DSS::OutputTab</class>
 <widget class="QWidget" name="DSS::OutputTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Output</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,10,20,20">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Output Files Settings</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QCheckBox" name="createOutput">
       <property name="text">
        <string comment="IDC_CREATEOUTPUT">Create Output file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="createHTML">
       <property name="text">
        <string comment="IDC_CREATEHTML">Create HTML Description file</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Output File Name</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="autoSave">
          <property name="text">
           <string comment="IDC_AUTOSAVE">Autosave.tif/fits</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="fileListName">
          <property name="text">
           <string comment="IDC_FILELIST">&lt;file list name&gt;.tif/fits</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="appendNumber">
          <property name="text">
           <string comment="IDC_APPENDNUMBER">Append a number to avoid file overwrite (001, 002, ...)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Output Location</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="refFrameFolder">
          <property name="text">
           <string comment="IDC_USEREFERENCEFRAMEFOLDER">Create Output file in the folder of the reference frame</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="fileListFolder">
          <property name="text">
           <string comment="IDC_USEFILELISTFOLDER">Create Output file in the folder of the file list</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,99">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="otherFolder">
          <property name="text">
           <string comment="IDC_USEANOTHERFOLDER">Create Output file in</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="outputFolder">
          <property name="styleSheet">
           <string notr="true">color: blue</string>
          </property>
          <property name="text">
           <string comment="IDC_OUTPUTFOLDER">&lt;Output Folder&gt;</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
